devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -174,6 +177,19 @@ private Map translateZoneStatus(ZoneStatus zs) {
     } else { }
 }
 
+private Map translateMultiStatus(String description) { 
+    def descMap = zigbee.parseDescriptionAsMap(description)
+	log.debug "descMap.value is : ${descMap.value}"
+    
```

Comment:
Again, `zigbee.parseDescriptionAsMap` is called on L109. You already have `descMap.value` when you call this function. Just pass the value directly instead of parsing the description again.